xrpl/core/binarycodec/types/issue.py

```diff
@@ -69,10 +78,16 @@ def from_parser(
         Args:
             parser: The parser to construct the Issue object from.
             length_hint: The number of bytes to consume from the parser.
+                For an MPT amount, pass 24 (the fixed length for Hash192).
 
         Returns:
             The Issue object constructed from a parser.
         """
+        if length_hint is not None and length_hint == 24:
```

Comment:
Suggestion: For purposes of readability, we can re-name this variable as:
`Hash192Size = 192 / 8`

Reply:
thanks